src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -328,13 +326,13 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
           </Box>
         </Box>
 
-        <RunWorkflowDialog
+        {/*<RunWorkflowDialog
```

Comment:
same as above.

Reply:
I will have follow up PRs to add them back. They are commented mainly for myself to make sure everything else builds